faq.html

```diff
@@ -0,0 +1,648 @@
+                          <strong>7. Where can I go if I have a question not during a lecture or lab hour?</strong>
+                      </button>
+                    </h1>
+                    <div id="collapseSeven" class="accordion-collapse collapse" aria-labelledby="headingSeven" data-bs-parent="#accordionExample">
+                      <div class="accordion-body" style="font-size:17px">
+                        Discord! All of the instructors will frequently check the OPS discord channel for any questions or comments from students. They will try and answer any questions they can virtually and give any guidance they can.
+                      </div>
+                    </div>
+                  </div>
+                  <div class="accordion-item">
+                    <h1 class="accordion-header" id="headingEight">
+                      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseEight" aria-expanded="false" aria-controls="collapseEight" style="font-size:20px">
+                          <strong>8. What is the time commitment each week for OPS?</strong>
+                      </button>
+                    </h1>
+                    <div id="collapseEight" class="accordion-collapse collapse" aria-labelledby="headingEight" data-bs-parent="#accordionExample">
+                      <div class="accordion-body" style="font-size:17px">
+                        It depends. If students slowly work on their projects through the two weeks, it may only take them an hour or two. However, each project can take up to 8 hours if attempted in a single sitting.
+                      </div>
+                    </div>
+                  </div>
+                  <div class="accordion-item">
+                    <h1 class="accordion-header" id="headingNine">
+                      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseNine" aria-expanded="false" aria-controls="collapseNine" style="font-size:20px">
+                          <strong>9. Can OPS help me in the future getting internships, research opportunities, or a job?</strong>
+                      </button>
+                    </h1>
+                    <div id="collapseNine" class="accordion-collapse collapse" aria-labelledby="headingNine" data-bs-parent="#accordionExample">
+                      <div class="accordion-body" style="font-size:17px">
+                        Absolutely! Companies love when applicants already have practical hands-on experience as well as self-made projects!
+                      </div>
+                    </div>
+                  </div>
+              </div>
+
+        </section>
+
+        
+        <!-- Section: Timeline -->
```

Comment:
Hi David. It looks like you copied over the timeline from the syllabus page. The elements are hidden from the viewport, but please remove the code anyway. It's extraneous.